actions.go

```diff
@@ -15,6 +15,52 @@ import (
 )
 
 // PipelineSaveAction creates the ActionFunc for saving pipeline configurations.
+func PipelineExecAction(clientConfig spinnaker.ClientConfig) cli.ActionFunc {
```

Comment:
I merged a doc PR just before yours, so this function needs a comment (and the one that's currently right above it needs to move to the function below it).